src/builder/webpack/config.browser.js

```diff
@@ -1,21 +1,36 @@
 // @flow
 
 import { join } from "path"
-
+import findCacheDir from "find-cache-dir"
 import commonWebpackConfig from "./config.common.js"
 import { offlinePlugin, offlineEntry } from "../../_utils/offline/webpack.js"
+import HardSourceWebpackPlugin from "hard-source-webpack-plugin"
 
 const chunkNameBrowser = "phenomic.browser"
 
 export default (config: PhenomicConfig): WebpackConfig => {
+  // Use different cache folder for hard-source-webpack-plugin on each env
+  const env = (config.production) ? "prod" : "dev"
```

Comment:
You can just use something like that https://www.npmjs.com/package/node-object-hash